tests/Kernel/fixtures/rendering.yml

```diff
@@ -515,7 +515,7 @@
       thumbnail:
         src: "https://inno-ecl.s3.amazonaws.com/media/examples/example-image.jpg"
         alt: "thumbnail alt"
-      resource_type: "resource type"
+      resource_type: "RESOURCE TYPE"
```

Comment:
This is supposed to be updated to upper case by the component.

Reply:
It's no longer the case so now it will be our responsibility to pass the formatter strings.

Reply:
Checked it with @22Alexandra and found out that, indeed the first element is made uppercase through CSS.

Reply:
However, since it is handled through CSS if we leave the string lowercase the test will fail since we test the raw rendered html.